oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/NimbusJwtDecoder.java

```diff
@@ -282,6 +284,20 @@ public JwkSetUriJwtDecoderBuilder cache(Cache cache) {
 			return this;
 		}
 
+		/**
+		 * Allows to customize the {@link JWTProcessor ConfigurableJWTProcessor} before passing it to
```

Comment:
Will you please reword this to match the JavaDoc for the other methods? Maybe something closer to:

```
Use the given {@link Consumer} to customize the {@link JWTProcessor ConfigurableJWTProcessor} before ...
```